### ISSUE TITLE
Get devices  RSSI(dmp) values 

### ISSUE THREAD

**Issue by erisanolasheni** [kiwi-bop/flutter_scan_bluetooth#3]
This is has been a very plugin that tends to support bluetooth scanning more the the popular flutter_blue library, however the only or part of the defects its having is that it is not giving the devices RSSI (dmp) values, developer needs this a lot to know the strength of the bluetooths available. Please will be happy if you can add this feature as soon as possible, this will give this library a high choice of preference.

Thank you.

**Comment by bilaalabdelhassan** [comment]
Hi, does it detect Android Mobile Phones? Flutter blue doesn't.

If it does, we would be glad to have the RSSI values implemented! Would be a great plus.

**Comment by erisanolasheni** [comment]
Yes sure. It does properly and it would be so great to implement RSSI values as soon as possible, thanks.

**Comment by jaumard** [comment]
I'm open to accept this as PR if you want to contribute but I'll not have time to do this any time soon

**Comment by erisanolasheni** [comment]
I did the Android version already, and not a Swift developer but I will try it, and make a PR, thanks.
